src/Collection.js

```diff
@@ -511,6 +511,10 @@ Collection.prototype.scroll = function (scrollId, options, filters, cb) {
     options = {};
   }
 
+  if (!options.scroll) {
+    throw new Error('Collection.scroll: scroll is required');
```

Comment:
`scroll` parameter is not required to do scroll request (that is only to force refreshing scroll id)